File: app/src/main/java/com/sagar/watchnext/network/newmodels/Image.java

package com.sagar.watchnext.network.newmodels;

import com.google.gson.annotations.SerializedName;

public class Image {

    @SerializedName("aspect_ratio")
    private float aspectRatio;

    @SerializedName("file_path")
    private String filePath;

    @SerializedName("height")
    private int height;

    @SerializedName("iso_639_1")
    private String iso6391;

    @SerializedName("vote_average")
    private float voteAverage;

    @SerializedName("vote_count")
    private int voteCount;

    @SerializedName("width")
    private int width;

    public float getAspectRatio() {
        return aspectRatio;
    }

    public String getFilePath() {
        return filePath;
    }

    public int getHeight() {
        return height;
    }

    public String getIso6391() {
        return iso6391;
    }

    public float getVoteAverage() {
        return voteAverage;
    }

    public int getVoteCount() {
        return voteCount;
    }

    public int getWidth() {
        return width;
    }
}

File: app/src/main/java/com/sagar/watchnext/network/newmodels/Image.kt
package com.sagar.watchnext.network.newmodels

import com.google.gson.annotations.SerializedName

class Image {

    @SerializedName("aspect_ratio")
    val aspectRatio: Float = 0.toFloat()

    @SerializedName("file_path")
    val filePath: String? = null

    @SerializedName("height")
    val height: Int = 0

    @SerializedName("iso_639_1")
    val iso6391: String? = null

    @SerializedName("vote_average")
    val voteAverage: Float = 0.toFloat()

    @SerializedName("vote_count")
    val voteCount: Int = 0

    @SerializedName("width")
    val width: Int = 0
}
